library/pkcs5.c

```diff
@@ -50,11 +50,12 @@
 #endif
 
 #include "hash_info.h"
+#include "mbedtls/psa_util.h"
 
 #if defined(MBEDTLS_ASN1_PARSE_C)
 static int pkcs5_parse_pbkdf2_params( const mbedtls_asn1_buf *params,
                                       mbedtls_asn1_buf *salt, int *iterations,
-                                      int *keylen, mbedtls_md_type_t md_type )
+                                      int *keylen, mbedtls_md_type_t* md_type )
```

Comment:
I don't understand why this change is needed or how it relates to the rest of the work. (Not saying it's wrong, just that I currently don't understand.)